runtime/ops/process.rs

```diff
@@ -301,37 +301,66 @@ pub fn kill(pid: i32, signal: &str) -> Result<(), AnyError> {
   use deno_core::error::type_error;
   use std::io::Error;
   use std::io::ErrorKind::NotFound;
-  use winapi::shared::minwindef::DWORD;
   use winapi::shared::minwindef::FALSE;
   use winapi::shared::minwindef::TRUE;
   use winapi::shared::winerror::ERROR_INVALID_PARAMETER;
   use winapi::um::errhandlingapi::GetLastError;
   use winapi::um::handleapi::CloseHandle;
   use winapi::um::processthreadsapi::OpenProcess;
   use winapi::um::processthreadsapi::TerminateProcess;
+  use winapi::um::wincon::GenerateConsoleCtrlEvent;
+  use winapi::um::wincon::CTRL_BREAK_EVENT;
+  use winapi::um::wincon::CTRL_C_EVENT;
   use winapi::um::winnt::PROCESS_TERMINATE;
 
-  if !matches!(signal, "SIGKILL" | "SIGTERM") {
-    Err(type_error(format!("Invalid signal: {}", signal)))
-  } else if pid <= 0 {
-    Err(type_error("Invalid pid"))
-  } else {
-    let handle = unsafe { OpenProcess(PROCESS_TERMINATE, FALSE, pid as DWORD) };
+  if pid <= 0 {
+    return Err(type_error(
+      "Process id (pid) cannot be negative or zero on Windows.",
```

Comment:
Wrt `CTRL_C_EVENT` and `CTRL_BREAK_EVENT`: Windows supports sending them to `pid 0`.

Reply:
TIL :) made some changes